app/views/profiles/show.html.erb

```diff
@@ -14,7 +12,7 @@
         <div id='friend-request-button' class='absolute -top-4 -right-2 '>
           <span class='bg-blue-200 py-1 px-2 rounded-full opacity-50'>Request Friend</span>
           <% if current_user == @user %>
-            <%= link_to 'Edit', edit_profile_path(@user), id: 'edit-profile-button', class: 'bg-blue-200 py-2 px-2 rounded-full w-9', data: { turbo_frame: 'edit-profile-modal' } %>
```

Comment:
the previous thing I mentioned is why this navigates via the unique_name, same thing